### PR TITLE
Fix small repo things

### PR DESCRIPTION
**Pull request by lucasfth**
- Add gitignore
- Remove DS_Store
- Fix Readme file type
- Add license

### CODE REVIEW

File: LICENSE
Comment (A-Guldborg) [inline LICENSE:1]: @PernilleK-ITU which license should we have?

File: .gitignore [hunk 1]
```diff
@@ -1,10 +1,40 @@
-# System files
-.DS_Store
-Thumbs.db
-
-# Parcel-bundler cache
-.cache/
-
-# Temp files
-tmp/
-temp/
```
Comment (A-Guldborg) [inline .gitignore:10]: Any particular reason you removed these? Are we sure no dependencies will create files of these types?

**Review by A-Guldborg — COMMENTED** [review]
Couple questions

File: .gitignore (hunk 1 above)
Comment (lucasfth) [reply .gitignore:10]: Dunno I will remove
(Actually think it was from another PR)

File: LICENSE
Comment (lucasfth) [reply LICENSE:1]: If I have made the wrong choice you could try out [repolicense](https://repolicense.com)

**Review by Scroll-Bar — APPROVED** [review]

**Pull request merged**